pipeline-scripts/launchdarkly-offboard.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -e
+shopt -s nocasematch
+window=5097615
+currenttime=$(date +%s)
+IFS=,
+
+function deleteuser() {
+        curl --request DELETE \
+          --url https://app.launchdarkly.com/api/v2/members/$1 \
+          --header 'Authorization: $TOKEN' \
+          --header 'LD-API-Version: beta'
+
+}
+curl -s --request GET   --url 'https://app.launchdarkly.com/api/v2/members?limit=200&offset=0'   --header "Authorization: $TOKEN"   --header 'LD-API-Version: beta' | jq -r '.items | map({_id, email, _lastSeen})| (first | keys_unsorted) as $keys | map([to_entries[] | .value]) as $rows | $keys,$rows[] | @csv' | tr -d \" | grep -v _lastSeen| while read id email lastseen
+do
+   # delete an user based on argument passed
+    if [[ $email == "$EMAIL" ]]
+    then
+      echo "Deleting user $EMAIL"
+      deleteuser $id
```

Comment:
I think we can just exit from here?
```suggestion
      deleteuser $id
      exit 0
```